yazi-plugin/src/external/fd.rs

```diff
@@ -16,7 +16,7 @@ pub fn fd(opt: FdOpt) -> Result<UnboundedReceiver<File>> {
 		.arg("--base-directory")
 		.arg(&opt.cwd)
 		.arg("--regex")
-		.args(if opt.hidden { ["--hidden", "--no-ignore"] } else { ["--no-hidden", "--ignore"] })
+		.args(if opt.hidden { ["--hidden"] } else { ["--no-hidden"] })
```

Comment:
```suggestion
		.arg(if opt.hidden { "--hidden" } else { "--no-hidden" })
```

Please use `arg()` instead. `rg.rs` is also.